src/crispy/StrongHash.h

```diff
@@ -21,11 +21,27 @@
 #include <string_view>
 #include <type_traits>
 
+#ifdef __x86_64__
 #include <immintrin.h>
 
 #if defined(__AES__)
     #include <wmmintrin.h>
 #endif
+#elif __aarch64__
```

Comment:
```suggestion
#elif defined(__aarch64__)
```
Shouldn't we test for "defined" rather then "evaluated to non-zero"?

Reply:
I copied that from here: https://github.com/f1ed/emp/blob/master/emp-tool/utils/block.h
You may be correct though.

Reply:
I this in the code tree overall you'll find `#if defined(...)` and `#elif defined(...)` the most. So keep it for now (Sadly, C++20 isn't even real-life applicable yet either. :) )